addons/im_livechat/static/src/core/thread_model_patch.js

```diff
@@ -34,6 +34,21 @@ patch(Thread.prototype, "im_livechat", {
         return this._super().filter((correspondent) => !correspondent.is_bot);
     },
 
+    get correspondent() {
+        let correspondent = this._super();
+        if (this.type === "livechat" && !correspondent) {
+            // For livechat threads, the correspondent is the first
+            // channel member that is not the operator.
+            const orderedChannelMembers = [...this.channelMembers].sort((a, b) => a.id - b.id);
+            correspondent =
+                this._store.self.id === orderedChannelMembers[0]?.persona.id &&
+                orderedChannelMembers[1]?.persona.type === this._store.self.type
+                    ? orderedChannelMembers[1]?.persona
+                    : orderedChannelMembers[0]?.persona;
+        }
```

Comment:
Why is the operator not used in the compute?

How does that compute from:
- the operator?
- the visitor?
- the invited person?

Reply:
Last version was not pushed, sorry. You are right, the check is done on the operator. If the first member is the operator, returns the next one. If not, return the first one.